tests/src/Kernel/ResolverBuilderTest.php

```diff
@@ -277,8 +277,11 @@ public function testDeferredCond() {
         return new Deferred(function () {
           return TRUE;
         });
-      }, $this->builder->fromValue('But this should.')
+        // Code Sniffer creates endless error loop trying to fix here.
```

Comment:
we don't run the fixer, we only run the checker. So that should not matter.

What is the PHPCS error message here? Can we fix it manually?

Reply:
[x] Array indentation error, expected 8 spaces but found 6
     |       |     (Drupal.Arrays.Array.ArrayIndentation)

wenn ich dann die Spaces auf 8 erhöhe kommt das

80 | ERROR | [x] Line indented incorrectly; expected 6 spaces, found 8
     |       |     (Drupal.WhiteSpace.ScopeIndent.IncorrectExact)
 280 | ERROR | [x] Closing brace indented incorrectly; expected 6 spaces,
     |       |     found 8 (Drupal.WhiteSpace.ScopeClosingBrace.Indent)

Dann ändere ich wieder zu 6 und das ganze beginnt wieder von vorne